lib/disability_compensation/factories/api_provider_factory.rb

```diff
@@ -153,6 +161,23 @@ def brd_service_provider
     end
   end
 
+  def generate_pdf_service_provider
+    case api_provider
+    when API_PROVIDER[:evss]
+      if @options[:auth_headers].nil? || @options[:auth_headers]&.empty?
```

Comment:
I think if @options[:auth_headers].blank? will cover both nil and empty scenarios here